src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/Opcode.cs

```diff
@@ -45,7 +45,13 @@ internal OpCode(OpCodeValues value, int flags)
         internal bool EndsUncondJmpBlk() =>
             (m_flags & EndsUncondJmpBlkFlag) != 0;
 
-        internal int StackChange() =>
+        /// <summary>
+        /// The value of how the IL instruction changes the evaluation stack.
+        /// </summary>
+        /// <remarks>
+        /// The difference between how many elements are popped from the stack and how many are pushed onto the stack as a result of the IL instruction.
```

Comment:
This needs to mention what this API returns when this is unknown (e.g. for `call` instruction)